README.md

```diff
@@ -71,3 +71,10 @@ console.log(id.toString()) // ... 4520031799277581759
 [codecov-url]: https://codecov.io/gh/googleapis/gcp-metadata
 [npm-image]: https://img.shields.io/npm/v/gcp-metadata.svg
 [npm-url]: https://www.npmjs.com/package/gcp-metadata
+
+### Set your own host with an environment variable
```

Comment:
I might just make this heading:

### Environment variables

* `GCE_METADATA_HOST`: provide an alternate host or IP to perform lookup against (useful, for example, you're connecting through a custom proxy server).
* `DETECT_GCP_RETRIES `:  number representing number of retries that should be attempted on metadata lookup.